### ISSUE TITLE
Test integration / setup flow

### ISSUE THREAD

**Issue by LaraMerdol** [iVis-at-Bilkent/software-artifact-analyzer#149]
@eraytuzun Please provide names of two who can do this.

**Comment by eraytuzun** [comment]
Utku Boran Torun boran.torun@ug.bilkent.edu.tr <boran.torun@ug.bilkent.edu.tr>
Selin Bahar Selin Bahar Gündoğar <bahar.gundogar@ug.bilkent.edu.tr>

**Comment by LaraMerdol** [comment]
### Instructions:
![Flow](https://github.com/user-attachments/assets/7c2b3317-177f-4235-b727-a450b1564f55)

**1.  Open Software Artifact Analyzer GitHub Application:**
Visit: [Software Artifact Analyzer](https://github.com/apps/software-artifact-analyzer)
Install it on your user account and select a specific project or grant permission to all projects.

**2.Redirect to SAA:**
You will be redirected to the SAA page.
Click on "Jira" to proceed to the Jira authentication page.

**3. Jira Permissions:**
Grant permission to the Jira instance you want to use with SAA.

**4.Neo4j Instance Selection:**
You will be redirected to the SAA Neo4j instance selection page.
You can continue with the default option or provide credentials for your own Neo4j instance.

**5. Access SAA:**
After selecting your Neo4j instance, you will be redirected to the SAA page.
From the top bar, navigate to Project -> New tab.

**6. Project Selection:**
You will see three select menus: one for selecting from example projects and one for selecting from your own projects.
Select a project from your personal projects.

**7. Initial Landing Comments (Optional):**
You have the option to add initial landing comments for pull requests in GitHub and issues in Jira.
If you check the corresponding boxes, initial landing comments will be added for your issues and pull requests.

**8.  Merge Similar Developers:**
You will be redirected to the "Merge Similar Developers" case.
Select and merge similar developers as needed.
In the "All Developers" section, you can merge developer groups that have not been merged yet.

**9. Filter Data:**
You can filter your data based on a specific time range.

**10. Construct Graph Data:**
Click "Confirm" to construct your graph data.
_**!!!!! If you are using the default Neo4j instance, ensure you are connected to the Bilkent VPN. !!!!**_

**Comment by torunb** [comment]
Hello,

I had a chance to test SAA (Software Artifact Analyzer). First, I downloaded the SAA to my GitHub account and connected it to my Jira account. I created a new project and then waited for the data to be retrieved. After selecting similarity groups, I chose the developers. After choosing developers, I skipped the date filtering and directly confirmed it.  While Lara was showing the app on Zoom, we figured there was a bug. She fixed it, and then I recreated the project. Since I had a problem with Bilkent VPN, I couldn't create the Neo4j Database, but Lara showed me the graph of my project. I think it is a very valuable project regarding productivity and efficiency. Thanks for everything. Elinize sağlık.
